### PR TITLE
First try at light/dark themes

### PR DESCRIPTION
**Pull request by jpyllman**
Will now use different theme depending on light or dark OS theme. Solves issue #6 .

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "manifest_version": 2,
   "name": "__MSG_extensionName__",
   "description": "__MSG_extensionDescription__",
-  "version": "2.0.0pre1",
+  "version": "2.0.0pre1+jpy13",
```
Comment (kewisch) [inline manifest.json]: Could you remove this from the PR?

File: popup/folderlist.js [hunk 2]
```diff
@@ -291,7 +296,7 @@ class TBFolderList extends HTMLElement {
     }
 
     this.selected = item;
-    this.shadowRoot.querySelector(".folder-list-body").focus();
+    //this.shadowRoot.querySelector(".folder-list-body").focus();
```
Comment (kewisch) [inline popup/folderlist.js]: Same

File: manifest.json (hunk 1 above)
Comment (jpyllman) [reply manifest.json]: OK, I will push a new commit where this is removed.

File: popup/folderlist.js (hunk 2 above)
Comment (jpyllman) [reply popup/folderlist.js]: This is the PR for the no focus. I will remove it from here and put it in its own PR.

**Pull request closed without merging**